feat(content): Lost Racer mission chain (#11267)
endless-sky/endless-sky: data/human/human missions.txt

			`	"I've got to speak with the race coordinators here," says Mr. Renard, striding towards the marshal station. "I see the marshals are back from searching. Syndicate idiots, probably didn't even notice she was missing till it was on the news. We've got to keep looking. Will you meet me back here in the spaceport in an hour or two? I'll double your fee."`
			choice
				`	"Of course. We're going to find her."`
					goto next
				`	"I'm sorry, but I can't help any more."`
			action
				set "abandoned lost racer"
			`	"Fine," he says, "then take your money." He slams a credit chip for <payment> in your hand and storms off into the spaceport, on the verge of tears.`
			`	Much later, you receive a request to meet him in the spaceport and take him to Glory.`
				decline
			label next
			`	"You're a saint for this," he says, and hands you a credit chip for <payment>. "We have to find her."`

mission "Lost Racer 2 - Abandoned"

> 	landing

Reviewer: This won't work as you intend it to, since the second mission is a landing mission and won't be offered from the spaceport. In addition I feel like there could be some other way to structure this. I'd move the choice to continue or to go to Glory to after you click spaceport, and then a mission would offer depending on whether you decide to take him to Glory or not. However, I don't know if there's a clean way to do this right now, I can only think of using the `entering` mission offer functionality in #11388.

Author: I think I've gotten it worked out, albeit in a manner slightly different to your suggestion.

				decline
			label greedy
			action
				payment 2000
			`	August turns. "What! After what you've done, crashed my hopes, you expect me to pay you! Why would you think that? Fine!" He throws a credit chip at you and exits <ship> in a huff.`
			`	When you access the credit chip, you find it is only worth <payment>.`

mission "Lost Racer 2"
	name "Search for Artemis Renard"
	description "Search the system of <system> for Artemis Renard, especially <planet>."
	source "Sunracer"
	waypoint "Acamar"
	destination "Sandswell"
	passengers 1
	deadline
	to offer
		has "Lost Racer 1: done"
		not "abandoned lost racer"
	on offer
		conversation
			`The displays in the spaceport hangar flicker with a news broadcast: sweeping aerial views of the spaceport alongside photos of Artemis Renard beside her red-and-white Flivver, wearing a calm, confident expression. Beneath the images, the screen reads "CORE CHAMPIONSHIP CATASTROPHE: MISSING MISS RENARD MYSTIFIES MILLIONS."`
			choice
				`	(Watch the news broadcast.)`
				`	(Wait for Mr. Renard to return.)`
					goto renard

> 			`	The voice of an eerily calm anchor plays over more vignettes of the race infrastructure. "... and, of course, today's top story: three-time Centurion-Division, Paradise Circuit racing champion Artemis Renard has been officially declared 'missing' after she did not return here to Sunracer after this year's Megaparsec Core Championship. The search effort has been ongoing for days now, led by Megaparsec with support from the Republic Navy and Syndicated Systems' internal security forces. The head of the event, Quinn Emerson, put out the following statement:`

Reviewer: ```suggestion
			`	The voice of an eerily calm anchor plays over more vignettes of the race infrastructure. "... and, of course, today's top story: three-time Centurion-Division, Paradise Circuit racing champion Artemis Renard has been officially declared 'missing' after she did not return here to Sunracer after this year's Megaparsec Core Championship. The search effort has been ongoing for days now, led by Megaparsec with support from the Republic Navy and Syndicated Security. The head of the event, Quinn Emerson, put out the following statement:`
```

			`	The broadcast then ends, going to an advertisement for the Megaparsec Splinter.`
			
			label renard
			`	Several minutes later, you see Mr. Renard returning from the terminal side of the spaceport, looking frustrated. "Idiots!" he shouts. "They couldn't find their legs in a dressing room! They tell me that they had 'some network and monitoring issues' and that they don't have surveillance for the entire last leg of the race. I told Artemis to never trust these Syndicate amateurs."`
			choice
				`	"I'm sure they're trying their best."`
				`	"'Network and monitoring issues' sounds suspicious."`
					goto suspicious
				`	"Do you have a plan for where to go next?"`
					goto plan

> 			`	"Well their best isn't very good, is it! They don't notice that their number-one pilot has been missing for days? I'll get the best lawyers in the Republic to drag Megaparsec through court 'till the only thing left they'll be able to sell will be scrap!"`

Reviewer 2: ```suggestion
			`	"Well their best isn't very good, is it! They don't notice that their number-one pilot has been missing for days? I'll get the best lawyers in the Republic to drag Megaparsec through court 'till the only thing left they'll be able to sell is scrap!"`
```

			choice
				`	"What you said about 'network and monitoring issues' sounded suspicious."`
				`	"So what's the plan from here?"`
					goto plan
			label suspicious
			`	Mr. Renard suddenly speaks more quietly. "I thought so too. I think it might have been sabotage."`
			`	"What do you mean?" you ask.`

> 			`	"Every day that Artemis is lost, the odds on her plummet and the bets on her competitors skyrocket. I've spend enough time making money to know that there's no faster way to bruise a competitor than scandal. I think someone sabotaged the race and went after my daughter because they knew it would make headlines, and they knew they could cash in by betting on underdogs."`

Reviewer 2: ```suggestion
			`	"Every day that Artemis is lost, the odds on her plummet and the bets on her competitors skyrocket. I've spent enough time making money to know that there's no faster way to bruise a competitor than scandal. I think someone sabotaged the race and went after my daughter because they knew it would make headlines, and they knew they could cash in by betting on underdogs."`
```

					goto abandon
				`	(Suggest a different system.)`
			action
				set "rescue artemis renard"
			`	"What about there," you say, pointing to the neighboring system. "Kugel. It's off the circuit, so no one will have searched there, but if her hyperdrive malfunctioned or was sabotaged, she might have ended up there. It's only a single jump away."`
			`	"Okay," he says, wiping his eyes with the corners of his sleeves. "Kugel it is, Captain."`
				accept
			
			label abandon
			`	"You're just like all the rest of those captains, don't care about their passengers, only seek to be paid." August Renard sighs. "But, I suppose that is to be expected. If nobody will help, giving up is the only option.`
			`	"At least take me back to Glory."`
				decline

	on enter "Kugel"
		dialog
			"As soon as you enter the Kugel system, your sensors begin to pick up a weak radio signal coming from the second planet, this time matching exactly the frequency of Artemis Renard's transponder. It seems you have arrived in the correct system."

mission "Lost Racer 3"
	landing
	name "Return Artemis Renard"
	description "You have found Artemis Renard. Transport her and her father back to <destination>."

> 	source "Sandswell"

Reviewer 2: Do you need to add an entirely new planet just for this mission?

			`	"You," the man says, "you're an independent captain, aren't you? You take missions for credits?" He winces a little bit after he says it, maybe a little embarrassed by his pointedness.`
			choice
				`	"I'm Captain <last>, and yes, I do."`
				`	"It depends on what the mission is."`
					goto depends
			`	"Oh. That's good to hear. I'm sorry for being so brusque with you, but my daughter's ship has gone missing. She's a racing pilot, best in the Paradise Circuit three times over; it's got to be some sort of foul play, pirates or a jealous competitor; that's what I told that useless Navy woman, that they've got to put out a search but she says that she has to have authorization..." He pauses for a moment, clearly realizing his digression.`

> 			`	"Again, I'm sorry. You don't need to hear all that. The loop was through Mirfak. I just need to know if you can take me to <destination> and scan for her transponder in those uninhabited systems north of there. Money is not an object."`
> 				goto help
> 			label depends
> 			`	He sighs. "It's search-and-rescue; my daughter's ship has gone missing. She's a racing pilot, best in the Republic three times over; it's got to be some sort of foul play, pirates or a jealous competitor; that's what I told that useless Navy woman, that they've got to put out a search but she says that she has to have authorization..." He pauses for a moment, clearly realizing his digression.`
> 			`	"Again, I'm sorry. You don't need to hear all that. The loop was through Mirfak. I just need to know if you can take me to <destination> and scan for her transponder in those uninhabited systems north of there. Money is not an object."`
> 			label help

Reviewer 3: We can get rid of a duplicate line this way, yes?
```suggestion
				goto help
			label depends
			`	He sighs. "It's search-and-rescue; my daughter's ship has gone missing. She's a racing pilot, best in the Republic three times over; it's got to be some sort of foul play, pirates or a jealous competitor; that's what I told that useless Navy woman, that they've got to put out a search but she says that she has to have authorization..." He pauses for a moment, clearly realizing his digression.`
			label help
			`	"Again, I'm sorry. You don't need to hear all that. The loop was through Mirfak. I just need to know if you can take me to <destination> and scan for her transponder in those uninhabited systems north of there. Money is not an object."`
```

				to display
					"total ships" > 1
			`	As you exit <ship>, you notice that the Flivver has been painted in the same colors that your ship has.`
			`	August approaches. "Artemis wanted to give you something as a present for saving her life, and we were wondering if you would like to get this Flivver. We've managed to refurbish it enough that it should be perfectly functional, and if you want a fast ship there's nothing in the sky that can beat you."`
			`	The Renards lead you up the steps into the palatial mansion and into a large whitewashed room with a golden table. On the table lie keys for the starship Artemis used in her most recent race.`
			action
				give ship "Racing Flivver" `B-168 "Moonfox"`
			`	"Well, I hope you like it." Artemis approaches. "It's been servicing me for quite a while, ever since I started racing. Unfortunately it's not fast enough anymore, even after the millions we spent on rebuilding it after the crash on Sandswell. Hopefully you'll be able to put it to better use." She smiles. "Of course, it would also be a good ship to sell, if you need the money."`

> 			`	You say your goodbyes and head back to the main spaceport.`

Reviewer 3: This struck me as a little brief; what do you think of this?
```suggestion
			`	After spending another half an hour or so catching up, you say your goodbyes to the pair and head back to the main spaceport.`
```

mission "Lost Racer 1"
	name "Search for Artemis Renard"

> 	description "Bring the father of a missing racing pilot to Sunracer by <date>, passing through the <waypoints> systems. You will be paid <payment>."

Author: ```suggestion
	description "Bring a missing racing pilot's father to <destination> by <date>, passing through the <waypoints> systems. You will be paid <payment>."
```

	minor
	passengers 1
	deadline
	source
		near "Kursa" 4
		attributes "paradise"
		not attributes "station" "tourism"
	waypoint "Volax"
	waypoint "Cardax"
	waypoint "Moktar"
	destination "Sunracer"
	to offer

> 		"combat rating" > 200
> 		random < 80

Reviewer 4: ```suggestion
		"combat rating" > 40
		random < 15
```
200 combat rating is probably a bit high, for reference marauder jobs can start appearing at 50 CR. Lowering the chance also makes it more consistent with other random missions.

Author: The issue here is that new players are unlikely to spend a long time in the Paradise Worlds, because they'll have to bribe to land.
I feel that the high random chance should be kept.

Reviewer 4: ```suggestion
		"combat rating" > 40
		random < 80
```
Sounds reasonable.

			`	"You," the man says, "you're an independent captain, aren't you? You take missions for credits?" He winces a little bit after he says it, maybe a little embarrassed by his pointedness.`
			choice
				`	"I'm Captain <last>, and yes, I do."`
				`	"It depends on what the mission is."`
					goto depends
			`	"Oh. That's good to hear. I'm sorry for being so brusque with you, but my daughter's ship has gone missing. She's a racing pilot, best in the Paradise Circuit three times over; it's got to be some sort of foul play, pirates or a jealous competitor; that's what I told that useless Navy woman, that they've got to put out a search but she says that she has to have authorization..." He pauses for a moment, clearly realizing his digression.`
				goto help
			label depends
			`	He sighs. "It's search-and-rescue; my daughter's ship has gone missing. She's a racing pilot, best in the Republic three times over; it's got to be some sort of foul play, pirates or a jealous competitor; that's what I told that useless Navy woman, that they've got to put out a search but she says that she has to have authorization..." He pauses for a moment, clearly realizing his digression.`
			label help

> 			`	"Again, I'm sorry. You don't need to hear all that. The loop was through Mirfak. I just need to know if you can take me to <destination> and scan for her transponder in those uninhabited systems north of there. Money is not an object."`

Author: "North" would refer to systems nearer to Alnitak than Earth. For someone who isn't a pilot, I don't think it's a stretch.

				goto board
			label fast
			`	"I'm glad you have pride in your ship. I've seen 'fast as they come' for twenty years now and I've never needed it as much as today."`
			label board
			`	You lead Mr. Renard to your ship, which he boards with unusual nimbleness and comfort for an old man from the Paradise Worlds.`
				accept
	on waypoint
		dialog "You have passed through all three systems laid out in the search plan, performing a long-range deep scan for Artemis Renard's transponder in each, but it has returned nothing but static. As far as you can tell, she is not in any of these systems."
	on complete
		payment 100000
		conversation
			`You were unable to find Artemis Renard in any of the three uninhabited systems, and you have watched the transition from hope to desperation in her father after each successive failure; you can't imagine that he has been getting much sleep.`

> 			`	During the landing on Sunracer, you notice that there is a great deal of race infrastructure still in place around the planet and spaceport: large camera drones remain in low orbit, and the spaceport is filled with a variety of Syndicate ship models in race-related livery. A small Navy contingent is also present, and the spaceport seems busier and more hectic than usual.`

Author: I think it'd be an "anything goes" race. How would you vary it from this definition?

> 			`	During the landing on Sunracer, you notice that there is a great deal of race infrastructure still in place around the planet and spaceport: large camera drones remain in low orbit, and the spaceport is filled with a variety of Syndicate ship models in race-related livery. A small Navy contingent is also present, and the spaceport seems busier and more hectic than usual.`

Reviewer 5: Got it, I think pretty good then, maybe tweak out Syndicate-specific then for the variety - the fastest ships would be ones like Flivvers, interceptors, even Shuttles. Maybe something like:

`...the spaceport is filled with a variety of ship models in racing livery, from tiny Shuttles to comparatively huge Splinters, and everything in between.`

That might be a tad wordy though, otherwise something like the below could work.
`...the spaceport is filled with a variety of ship models in racing livery.`

	source "Sunracer"
	destination "Pummel"
	passengers 1
	deadline
	to offer
		has "Lost Racer 1: done"
		not "abandoned lost racer"
	on offer
		conversation
			`The displays in the spaceport hangar are playing a news channel, showing aerial views of the spaceport and pictures of the missing Artemis Renard in front of her red and white Flivver, wearing a calm, confident expression while a caption at the bottom of the screen reads "CHAMPION RACER MISSING AFTER COMPETITION."`
			choice
				`	(Watch the news broadcast.)`
				`	(Wait for Mr. Renard to return.)`
					goto renard
			`	The voice of an eerily calm anchor plays over more vignettes of the race infrastructure. "... and, of course, today's top story: three-time Republic starship racing champion Artemis Renard has been officially declared 'missing' after she did not return here to Sunracer after this year's Megaparsec Core Championship. The search effort has been ongoing for two days now, led by Megaparsec with support from the Republic Navy and Syndicated Systems' internal security forces. The head of the event, Quinn Emerson, put out the following statement:`

> 			`	'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`

Author: It's unclear. I don't think it's implausible that they'd have some level of control, even if they don't have full jurisdiction.

> 			`	'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`

Reviewer 5: I don't think it'd be that implausible either, and I think it would probably be fine. Though, it does feel like a change to me vs how they've previously been portrayed. Perhaps we wait to see if others have thoughts?

> 			`	'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`

Reviewer: I think it's fine, although I imagine the Republic (at least used to) have more control over the uninhabited systems given the old Navy base in Nihal. I don't see why they wouldn't be allowed to scan Syndicate systems if they were a part of it.

			scene "scene/flivver crash"
			`	Mr. Renard shouts with what you believe to be joy at seeing the parachute and scattered flares. The two of you wade through the sand and increasingly dense scattered debris as quickly as you can until you arrive at the rear hatch of the crashed Flivver. You struggle to turn the lock but eventually force it open, briefly letting the sand-filled air rush into the interior of the ship before you close it behind you. Inside, the ship is dark and depowered but more or less intact. At the front of the craft, sitting in the half-submerged cockpit, is the woman you saw on the broadcast, wiping the sand from her face.`
			`	"Hmph," she says, somewhat weakly. "I thought you were going to be the Navy."`
			`	Mr. Renard rushes to her and the two embrace.`
			`	"And who's this?" Artemis asks.`
			choice
				`	"Captain <first> <last> of the <ship>."`
				`	"I'm here to get you out of here."`
					goto notime
			label introduce
			`	"A pilot, eh? I can appreciate that. I'd appreciate it more if you could fly me out of here, though."`
			choice
				`	"My ship is landed outside. Follow me."`
					goto ship
				`	"I have some questions."`

> 			`	Artemis coughs. "Later, okay?"`

Author: Isn't that addressed by the `label galley` questions?

> 			`	Artemis coughs. "Later, okay?"`

Reviewer 5: Kind of, because when you later ask the `label galley` questions, they once again tell you they'll talk about it later. 

If the "Ah well, that's probably something to talk about later" gets a resolution there, then I think this comment can be resolved since a solution to that would address this one as well.

So feel free to resolve this one to consolidate with that other comment.

			`	You don't think you've ever seen someone devour a sleeve of freeze-dried peas as quickly or with as much verve as Artemis does. After eating her way through several more of the <ship>'s varieties of stored foodstuffs, she eventually slows down, though the hungry look on her face remains. Her father sits beside her, tightly holding her hand.`
			`	"I suppose you have some questions," she says, "both of you. Well, I hope you do, at least."`
			choice
				`	"Why did you crash?"`
				`	"That was a beautiful ship you had there. I've never seen a Flivver quite like that."`
					goto beautiful
				`	"I'll leave the questions to your father."`
					goto noquestions
			label crash
			`	"I really do not know. We checked everything before launch, everything, and yet during the jump the Moonfox' hyperdrive malfunctioned, knocking me out. When I woke up, the Flivver was damaged and I was stranded on a planet with no way of calling for help - the flares had triggered when I crashed. Well, now you know. Is there anything else?"`
			`	August asks, "Do you think someone could have sabotaged it? Any suspicious Syndicate employees hanging around? Any anything out of the ordinary?`

> 			`	"Ah well, that's probably something to talk about later."`

Reviewer 2: I agree, it's weird that the entire potential conspiracy plot thread just gets dropped and never addressed again.

		conversation
			`As the advanced medical facilities of <planet> rush up and past the ship, August comes to the bridge. "Truly, thank you, Captain, for sticking through this mess with me, and for continuing to search even when hope was gone.`
			`	"I would like to reward you with a payment of <payment> for your services, and apologize for the inconvenience this might have caused you."`
			`	Soon afterwards, the Renards head out of the <ship>. The moment they exit the hatch, a veritable army of attendants appears out of nowhere, ready to help them move the five feet to a Shuttle that has just landed on a pad next to you. August waves, and steps in.`

event "artemis renard recovered"

mission "Lost Racer Epilogue"
	landing
	name "Visit Artemis Renard"
	description `Retrieve some "surplus equipment" from Artemis Renard on <destination>.`
	source
		near Sol 1 100
	destination "Glory"
	to offer
		has "event: artemis renard recovered"
	on offer
		conversation
			`While you are checking your incoming messages, you notice one from Artemis Renard, the crashed racing pilot who you rescued a couple months ago.`
			`	"Captain <last>,`

> 			`	"I'd like to thank you again for your timely rescue. Recovering from the ordeal has been a long process, of course, but a worthwhile one. Unfortunately that length of time has meant that following the settlement with Megaparsec and as I've been working my way back into the various circuits, we've built up a surplus of increasingly outdated equipment. While we could, of course, simply have it scrapped locally, we were hoping that you might join us on <destination> to see if you can put it to better use."`

Reviewer 5: Why would Megaparsec pay out when the investigation would reveal that it was due to a malfunction of Artemis' equipment (Hyperdrive)? If anything, at most they'd be able to sue their ship/team, but that seems to be themselves so I don't know if that would work out. Can just take out that one line and it still works well:
```suggestion
			`	"I'd like to thank you again for your timely rescue. Recovering from the ordeal has been a long process, of course, but a worthwhile one. As I've been working my way back into the various circuits, we've built up a surplus of increasingly outdated equipment. While we could, of course, simply have it scrapped locally, we were hoping that you might join us on <destination> to see if you can put it to better use."`
```

This could also be a natural spot to ask the "questions late" mentioned earlier.

			`		Sincerely,`
			`			Artemis Renard`
				accept
	on complete
		log "Was given a Racing Flivver as additional compensation for helping find Artemis Renard."
		conversation
			`As Glory again rises to greet your ship, you receive a hail from a spacecraft that comes up alongside your ship. "Ho, Captain! Why don't you come to our estate, instead of the main landing port? It'll be easier." August Renard is leaning out the cockpit of a fancy Racing Flivver, looking much happier than when you first met him.`
			`	You redirect your ship in the direction the Flivver is going, noticing that it has the same name as Artemis' ship which crashed on Sandswell.`

> 			`	Soon, the dense main city of <planet> gives way to a perfectly-terraformed countryside with only one mar - a gigantic mansion that must be hundreds of meters high. A few landing pads are set in a neat array next to it, and the B-168 "Moonfox" lands on one of these, beckoning for you to land on another.`

Reviewer 5: The tallest personal residence in the world is [a personal skyscraper 173 meters tall](https://en.wikipedia.org/wiki/Antilia_(building)). Mansions tend to sprawl more than they go up vertically, so a mansion hundreds of meters high evokes something that ought to be even wider, which evokes a sprawling mansion scaled up to the height of a tall skyscraper, which is a bit much. Swapping over to be a personal skyscraper fixes that, or alternatively could be tweaked to something like `sprawling mansion covering thousands of square meters`.

```suggestion
			`	Soon, the dense main city of <planet> gives way to a perfectly-terraformed countryside with only one mar - a gigantic personal skyscraper that must be hundreds of meters high. A few landing pads are set in a neat array next to it, and the B-168 "Moonfox" lands on one of these, beckoning for you to land on another.`
```

			`	As you exit <ship>, you notice that the Flivver has been painted in the same colors that your ship has.`
			`	August approaches. "Artemis wanted to give you something as a present for saving her life, and we were wondering if you would like to have this Flivver. We've managed to refurbish it enough that it should be perfectly functional, and if you want a fast ship there's nothing in the sky that can beat you."`
			`	August leads you up the steps into the palatial mansion and into a large whitewashed room with a golden table. On the table lie keys for the starship Artemis used in her most recent race.`
			action
				give ship "Racing Flivver" `B-168 "Moonfox"`
			`	"Well, I hope you like it." Artemis approaches. "It's been servicing me for quite a while, ever since I started racing. Unfortunately it's not fast enough anymore, even after the millions we spent on rebuilding it after the crash on Sandswell. Hopefully you'll be able to put it to better use." She smiles. "Of course, it would also be a good ship to sell, if you need the money."`
			`	After spending another half an hour or so catching up, you say your goodbyes to the pair and head back to the main spaceport.`

> ship "Racing Flivver"

Reviewer 5: Neat ship, and very pretty! Have not done a balance review yet - will probably wait until we get a confirmation from devs on whether adding this as a one off ship is desired or not.

			choice
				`	(Watch the news broadcast.)`
				`	(Wait for Mr. Renard to return.)`
					goto renard
			`	The voice of an eerily calm anchor plays over more vignettes of the race infrastructure. "... and, of course, today's top story: three-time Centurion-Division, Paradise Circuit racing champion Artemis Renard has been officially declared 'missing' after she did not return here to Sunracer after this year's Megaparsec Core Championship. The search effort has been ongoing for days now, led by Megaparsec with support from the Republic Navy and Syndicated Security. The head of the event, Quinn Emerson, put out the following statement:`
			`	'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`

> 			`	'Artemis Renard, daughter of terraforming magnates August and Louisa Renard, is perhaps best known for shattering the record for the Betelgeuse Classic in last year's race, crossing the finish line nearly fifteen minutes faster than the record set by Jackson Lee over twenty years ago. She was last seen flying her ship, the "Moonfox," a white and red Lionheart Flivver with race registration B-168, in the Diphda system. Anyone with information regarding her whereabouts is encouraged to contact Syndicate or Navy authorities, as well as the Megaparsec Racing Events Commission, and may be entitled to monetary reward if such information leads to the retrieval of Ms. Renard or her ship.'"`

Author: That's the way it _should_ work, but the Content Style check doesn't seem to like it.

> 			`	'Artemis Renard, daughter of terraforming magnates August and Louisa Renard, is perhaps best known for shattering the record for the Betelgeuse Classic in last year's race, crossing the finish line nearly fifteen minutes faster than the record set by Jackson Lee over twenty years ago. She was last seen flying her ship, the "Moonfox," a white and red Lionheart Flivver with race registration B-168, in the Diphda system. Anyone with information regarding her whereabouts is encouraged to contact Syndicate or Navy authorities, as well as the Megaparsec Racing Events Commission, and may be entitled to monetary reward if such information leads to the retrieval of Ms. Renard or her ship.'"`

Reviewer 6: That seems like an issue with the content style check then, probably...

			`	"Oh, shove off, Captain. I'm fine. Automated drogue system in the ship, safety first, spare no expense. Wouldn't mind a visit to your galley, though."`
			choice
				`	"Of course."`
					goto galley
				`	"Drogue system? Like parachutes? I hadn't considered that."`
			`	"Really? What is it with you spacers and 'going down with the ship' anyway?"`
			label galley
			`	You don't think you've ever seen someone devour a sleeve of freeze-dried peas as quickly or with as much verve as Artemis does. After eating her way through several more of the <ship>'s varieties of stored foodstuffs, she eventually slows down, though the hungry look on her face remains. Her father sits beside her, tightly holding her hand.`
			`	"I suppose you have some questions," she says, "both of you. Well, I hope you do, at least."`
			choice
				`	"Why did you crash?"`
				`	"That was a beautiful ship you had there. I've never seen a Flivver quite like that."`
					goto beautiful
				`	"I'll leave the questions to your father."`
					goto noquestions
			label crash

> 			`	"I really do not know. We checked everything before launch, everything, and yet during the jump the Moonfox's hyperdrive malfunctioned, knocking me out. When I woke up, the Flivver was damaged and I was stranded on a planet with no way of calling for help - the flares had triggered when I crashed. Well, now you know. Is there anything else?"`

Reviewer 6: Okay...
1. What are the odds that a hyperdrive malfunction would send you to some other place? I'd assume almost zero, and that the likely result would be total destruction.
2. How hard would it be to make some change to a ship's firmware that triggers a hyperspace jump in a specific direction when certain conditions are met and makes it look like a malfunction? Would that be possible to do in a way that fools one of the most skilled racing pilots in the galaxy?

Author: 1. Fair.
2. I'm not an expert in hyperdrive programming, but I don't think it would be too difficult, if a possible enemy was willing to spend a lot. If it was possible to fool Artemis, August would trust her opinion and decide that a conspiracy wouldn't have happened.

Reviewer 2: Adding onto this - what are the odds that the Moonfox, after a hyperdrive malfunction where the pilot is unconscious, would conveniently crash land on the only landable planet in the system?
Perhaps it would make more sense if the Moonfox's engines had malfunctioned, and Artemis attempted an emergency landing on Sandswell?

Author: The question is, what makes a planet landable? In theory, all planets are landable; we just haven't added data for them. Sandswell is a landable because that's where Artemis crashes, but if she had crashed on a different planet then that would have been the one with data that the player could land on.
An emergency landing could work.

Reviewer 2: I meant more along the lines of why would she have crashed into a planet at all? Space is big, in all likelihood her ship would've ended up floating through space if her hyperdrive malfunctioned and she was knocked unconscious.

			`	You redirect your ship in the direction the Flivver is going, noticing that it has the same name as Artemis' ship which crashed on Sandswell.`
			`	Soon, the dense main city of <planet> gives way to a perfectly-terraformed countryside with only one mar - a gigantic personal skyscraper that must be hundreds of meters high. A few landing pads are set in a neat array next to it, and the B-168 "Moonfox" lands on one of these, beckoning for you to land on another.`
			choice
				`	(Land on one of the pads.)`
					goto continue
				`	"What about my fleet?"`
					to display
						"total ships" > 1
					goto continue
				`	"You have a mansion here!"`
			`	"Later, later," Artemis shouts. "We'll talk in a place where we needn't shout."`
			label continue
			`	August calls, "You can let your fleet stay at the main spaceport; I'm afraid there's not room here."`
				to display
					"total ships" > 1

> 			`	As you exit the <ship>, you notice that the Flivver has been painted in the same colors that your ship has.`

Reviewer 6: ```suggestion
			`	As you exit the <ship>, you notice that the Flivver has been painted in the same colors as your ship.`
```

			label help
			`	"Again, I'm sorry. You don't need to hear all that. The loop was through Mirfak. I just need to know if you can take me to <planet> and scan for her transponder in those uninhabited systems north of there. Money is no object."`
			choice
				`	"I'd be happy to help."`
					goto agree
				`	"I'm sorry, but I can't transport you. You'll have to find another captain."`
			`	A flicker of emotions crosses the man's face: anger, scorn, resignation. "I thought you'd be better than those Navy bastards," he says, shaking his head. "Well, I hope I can find someone in this place with a shred of decency before she dies out there." He walks away, his disappointment heavy in the air.`
				decline
			label agree
			`	The man exhales, relieved. "Thank you so much," he says. "My name is August Renard, and my daughter is Artemis. She's out there, I know it, but we'd better get moving soon."`
			choice
				`	"My ship is as fast as they come. We'll find her in no time."`
					goto fast
				`	"Artemis Renard? I think I heard she won something recently... maybe in Betelgeuse?"`

> 			`	"Ah, yes," the man says with pride in his voice. "The Betelgeuse Classic, a circuit of every planet and major moon in the system, and hitting every marker in an equatorial orbit without refueling. They've been doing it since the Firebird was new, and no one's done it faster than my daughter - no one. She's the best; that's how I know something's gone wrong. Tell your people to warm up the engines."`

Author: ```suggestion
			`	"Ah, yes," the man says with pride in his voice. "The Betelgeuse Classic. Contestants run a circuit of every planet and major moon in the system and hit every marker in an equatorial orbit without refueling. They've been doing it since the Firebird was new, and no one's done it faster than my daughter - no one. She's the best; that's how I know something's gone wrong. Tell your people to warm up the engines."`
```

			choice
				`	"I guess."`
					accept
				`	"Sorry. You'll have to find another captain."`
					decline
	on complete
		log "Conducted a brief search but was unable to find anything of the famous racing pilot Artemis Renard. Returned her grieving father home to Glory."
		log "Minor People" "August and Artemis Renard" `August Renard contracted me to help look for his daughter Artemis, a racing pilot who went missing. Unable to find the woman, but returned her father home to Glory.`
		conversation
			`As the advanced medical facilities of <planet> rise up to greet the <ship>, August comes to meet you on the bridge. "Captain, I would like to thank you for attempting to find my daughter, even though the attempt ended poorly. Thank you for that, at least." He then prepares to exit the ship, outside of which a veritable army of attendants are waiting.`
			choice
				`	"Weren't you going to pay me?"`
					goto greedy
				`	(Leave him alone.)`
			`	August is escorted to a Shuttle which has just landed on the pad next to the <ship>, waves to you as he goes in, and launches to another part of the planet.`
				decline
			label greedy
			action
				payment 2000

> 			`	August turns. "What! After what you've done, crashed my hopes, you expect me to pay you! Why would you think that? Fine!" He throws a credit chip at you and exits the <ship> in a huff.`

Author: I think using an exclamation mark here conveys August's anger better.

> 			`	August turns. "What! After what you've done, crashed my hopes, you expect me to pay you! Why would you think that? Fine!" He throws a credit chip at you and exits the <ship> in a huff.`

Author: I changed the other one.

	source "Sunracer"
	waypoint "Acamar"
	destination "Sandswell"
	passengers 1
	deadline
	to offer
		has "Lost Racer 2 - Ended: declined"
		not "abandoned lost racer"
	on offer
		conversation
			`The displays in the spaceport hangar flicker with a news broadcast: sweeping aerial views of the spaceport alongside photos of Artemis Renard beside her red-and-white Flivver, wearing a calm, confident expression. Beneath the images, the screen reads "CORE CHAMPIONSHIP CATASTROPHE: MISSING MISS RENARD MYSTIFIES MILLIONS."`
			choice
				`	(Watch the news broadcast.)`
				`	(Wait for Mr. Renard to return.)`
					goto renard
			`	The voice of an eerily calm anchor plays over more vignettes of the race infrastructure. "... and, of course, today's top story: three-time Centurion-Division, Paradise Circuit racing champion Artemis Renard has been officially declared 'missing' after she did not return here to Sunracer after this year's Megaparsec Core Championship. The search effort has been ongoing for days now, led by Megaparsec with support from the Republic Navy and Syndicated Security. The head of the event, Quinn Emerson, put out the following statement:`

> 			`	"'We at Megaparsec put the greatest attention into ensuring the safety and well-being of all our event participants and guests. Over the past forty-six hours, our search teams have fully scanned the Volax, Cardax, Moktar, Bellatrix, and Menkar systems, and we are working in conjunction with the Republic and Syndicate to fully scour their territories belonging to the circuit as well. Ms. Renard's family can rest assured that if she's out there, we'll find her.'`

Author: I added the extra quotation mark at mOctave's suggestion. Is there a reason for removing it again?

> 			`	"'Artemis Renard, daughter of terraforming magnates August and Louisa Renard, is perhaps best known for shattering the record for the Betelgeuse Classic in last year's race, crossing the finish line nearly fifteen minutes faster than the record set by Jackson Lee over twenty years ago. She was last seen flying her ship, the "Moonfox," a white and red Lionheart Flivver with race registration B-168, in the Diphda system. Anyone with information regarding her whereabouts is encouraged to contact Syndicate or Navy authorities, as well as the Megaparsec Racing Events Commission, and may be entitled to monetary reward if such information leads to the retrieval of Ms. Renard or her ship.'"`

Reviewer 6: This is just as correct if not more so and might appease stylecheck?

```suggestion
			`	"'Artemis Renard, daughter of terraforming magnates August and Louisa Renard, is perhaps best known for shattering the record for the Betelgeuse Classic in last year's race, crossing the finish line nearly fifteen minutes faster than the record set by Jackson Lee over twenty years ago. She was last seen flying her ship, the Moonfox, a white and red Lionheart Flivver with race registration B-168, in the Diphda system. Anyone with information regarding her whereabouts is encouraged to contact Syndicate or Navy authorities, as well as the Megaparsec Racing Events Commission, and may be entitled to monetary reward if such information leads to the retrieval of Ms. Renard or her ship.'"`
```

mission "Lost Racer 2 - Failed"
	entering
	name "Take August to <planet>"
	description "After your failed search for Artemis Renard, take her father, August, back to <destination>."
	source
		system "Acamar"
	destination "Glory"
	passengers 1
	to offer
		has "Lost Racer 2 - Continued: failed"
		has "lost racer august to glory"
	on complete
		clear "lost racer august to glory"
		conversation
			`As the advanced medical facilities of <planet> rise up to greet the <ship>, August comes to meet you on the bridge. "Captain, I would like to thank you for attempting to find my daughter, even though the attempt ended poorly. Thank you for that, at least." He then prepares to exit the ship, outside of which a veritable army of attendants are waiting.`
			choice
				`	"Weren't you going to pay me?"`
					goto greedy
				`	(Leave him alone.)`
			`	August is escorted to a Shuttle which has just landed on the pad next to the <ship>, waves to you as he goes in, and launches to another part of the planet.`
				decline
			label greedy
			action
				payment 2000

> 			`	August turns. "What? After what you've done, crashed my hopes, you expect me to pay you? Why would you think that? Fine!" He throws a credit chip at you and exits the <ship> in a huff.`

Author: ```suggestion
			`	August turns. "What! After what you've done, crashed my hopes, you expect me to pay you? Why would you think that? Fine!" He throws a credit chip at you and exits the <ship> in a huff.`
```

			`	"Oh, shove off, Captain. I'm fine. Automated drogue system in the ship, safety first, spare no expense. Wouldn't mind a visit to your galley, though."`
			choice
				`	"Of course."`
					goto galley
				`	"Drogue system? Like parachutes? I hadn't considered that."`
			`	"Really? What is it with you spacers and 'going down with the ship' anyway?"`
			label galley
			`	You don't think you've ever seen someone devour a sleeve of freeze-dried peas as quickly or with as much verve as Artemis does. After eating her way through the <ship>'s entire stock of several other varieties of packaged food, she eventually slows down, though the hungry look on her face remains. Her father sits beside her, holding her hand tightly.`
			`	"I suppose you have some questions," she says, "both of you. Well, I hope you do, at least."`
			choice
				`	"Why did you crash?"`
				`	"That was a beautiful ship you had there. I've never seen a Flivver quite like that."`
					goto beautiful
				`	"I'll leave the questions to your father."`
					goto noquestions
			label crash

> 			`	"I really do not know. We checked everything before launch, everything, and yet during the jump the Moonfox's hyperdrive malfunctioned, knocking me out. When I woke up, the Flivver was damaged and hurtling toward outer space. Luckily, the engines hadn't been damaged too much, so I was able to turn around and thrust in the direction of <origin>, but then the batteries gave out and the generator wasn't working. So I was stranded on a planet with no way of calling for help - the flares had triggered when I crashed, being unable to slow down without thrusters. Well, now you know. Is there anything else?"`

Author: ```suggestion
			`	"I really do not know. We checked everything before launch, everything, and yet during the jump the Moonfox's hyperdrive malfunctioned, knocking me out. When I woke up, the Flivver was damaged and hurtling toward outer space. Luckily, the engines hadn't been harmed too much, so I was able to turn around and thrust in the direction of <origin>, but then the batteries gave out and the generator stopped working. So I ended up stranded on a planet with no way of calling for help - the flares had triggered when I crashed, being unable to slow down without thrusters. Well, now you know. Is there anything else?"`
```
